### ISSUE TITLE
slow deferred open

### ISSUE THREAD

**Issue by mcollina** [Level/levelup#190]
The deferred open is slow, because of the call to `once` in here (https://github.com/rvagg/node-levelup/blob/master/lib/levelup.js#L49-L58).

Thanks to @trevornorris, we got this amazing data:

```
Ticks per symbol:
    164979  60.0% LazyCompile:*EventEmitter.removeListener events.js:200 [js]
     22855   8.3% Stub:CompareICStub [js]
      1940   0.7% v8::internal::JSObject::SetPropertyForResult(...

Stats:
    275340 total trace events
    275095 total ticks
     16910 ticks not in symbols
     13184   4.8% ticks in unaccounted ticks (probably in the kernel, try --kernel)
    167521  60.9% ticks in ticks in optimized code
      2285   0.8% ticks in ticks in other lazily compiled code
     44003  16.0% ticks in ticks in v8::internal::*
    135143 total symbols
      2693 used symbols
     6.97s library processing time
   130.44s tick processing time
```

Here is the bench:

```
var db = require('levelup')('./mydb');
var val = new Buffer(100).fill('bar');
var iter = 1e4;


for (var i = 0; i < iter; i++)
  db.put(Math.random(), val, function(err) {
    if (err)
      throw new Error(err);
  });
```

The suggestion from @trevornorris is to have a single queue for every methods and call `once` only once.
